Makefile

```diff
@@ -129,24 +132,21 @@ run_debian: salus
 		-device guest-loader,kernel=$(LINUX_BIN),addr=$(KERNEL_ADDR) \
 		-append "$(BOOTARGS) root=LABEL=rootfs" \
 		-device guest-loader,initrd=$(INITRD_IMAGE),addr=$(INITRD_ADDR) \
-		-device x-riscv-iommu-pci \
 		-drive file=$(ROOTFS_IMAGE),if=none,id=hd \
-		-device nvme,serial=deadbeef,drive=hd \
-		-netdev user,id=usernet,hostfwd=tcp:127.0.0.1:7722-0.0.0.0:22 \
-		-device e1000e,netdev=usernet \
+		-$(NVME_DEVICE_ARGS) \
+		-$(IOMMU_ARGS) \
+		-$(NETWORK_ARGS) \
 		$(EXTRA_QEMU_ARGS)
 run_buildroot: salus
 	$(QEMU_BIN) \
 		$(MACH_ARGS) \
 		-kernel $(RELEASE_BINS)salus \
 		-device guest-loader,kernel=$(LINUX_BIN),addr=$(KERNEL_ADDR) \
 		-append "$(BUILDROOT_BOOTARGS)" \
-		-drive file="$(BUILDROOT_INITRD_IMAGE),format=raw,id=hd0" \
```

Comment:
s/hd0/hd/ is not mentioned in the commit message. Should that be in a prior commit?

Reply:
Squashed the PATCH 1 & 4 now.